device/globals/esc_compile.gd

```diff
@@ -256,21 +262,12 @@ func read_dialog_option(state, level, errors):
 	var cmd = { "name": "*", "params": [question, []] }
 
 	if q_flags:
-		#printt("parsing flags ", q_flags, state.line)
-		var if_true = []
-		var if_false = []
-		var if_inv = []
-		var if_not_inv = []
+		var ifs = {"if_true": [], "if_false": [], "if_inv": [], "if_not_inv": [], "if_eq": [], "if_ne": []}
 		var flag_list = []
-		parse_flags(q_flags, flag_list, if_true, if_false, if_inv, if_not_inv)
-		if if_true.size():
-			cmd.if_true = if_true
-		if if_false.size():
-			cmd.if_false = if_false
-		if if_inv.size():
-			cmd.if_inv = if_inv
-		if if_not_inv.size():
-			cmd.if_not_inv = if_inv
```

Comment:
I think this must have been a bug, but since the `inv-inv_item` and `!inv-inv_item` is not documented anywhere, as far as I know, it was probably never encountered.